package.json

```diff
@@ -34,9 +33,12 @@
     "@loopback/authentication": "^4.0.0-alpha.5",
     "@loopback/context": "^4.0.0-alpha.11",
     "@loopback/core": "^4.0.0-alpha.11",
+    "@loopback/repository": "^4.0.0-alpha.7.tgz",
```

Comment:
Why `^4.0.0-alpha.7.tgz`?

Reply:
Ah, I was installing locally from a file and incorrectly changed the version spec for this commit.